FLEDGE.md

```diff
@@ -874,6 +874,10 @@ The arguments to `generateBid()` are:
 
 *   interestGroup: The interest group object, as saved during `joinAdInterestGroup()` and perhaps updated via the `updateURL`.
     * `priority` and `prioritySignalsOverrides` are not included. They can be modified by `generatedBid()` calls, so could theoretically be used to create a cross-site profile of a user accessible to `generateBid()` methods, otherwise.
+    * `lifetimeMs` is not included. It's ambiguous what should be passed: the lifetime when the group was joined,
+      or the remaining lifetime. Providing the remaining lifetime would also potentially give access to more
```

Comment:
The other blocks don't include line breaks.  This does.  Is this a deliberate style change?  Otherwise, looks good.

Reply:
Yes, I'm deliberately line wrapping new sections.  GitHub's code review tool seems to get confused by long lines and fails to show the individual changed words in the line.  If I can get the open pull request count down low enough, I may line wrap the whole document, otherwise it'll cause a lot of merge conflicts.